lib/ansible/modules/remote_management/redfish/redfish_info.py

```diff
@@ -0,0 +1,401 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2017-2018 Dell EMC Inc.
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: redfish_info
+version_added: "2.9"
+short_description: Manages Out-Of-Band controllers using Redfish APIs
+description:
+  - Builds Redfish URIs locally and sends them to remote OOB controllers to
+    get information back.
+  - Information retrieved is placed in a location specified by the user.
+options:
+  category:
+    required: false
+    description:
+      - List of categories to execute on OOB controller
+    default: ['Systems']
+    type: list
+  command:
+    required: false
+    description:
+      - List of commands to execute on OOB controller
+    type: list
+  baseuri:
+    required: true
+    description:
+      - Base URI of OOB controller
+    type: str
+  username:
+    required: true
+    description:
+      - User for authentication with OOB controller
+    type: str
+    version_added: "2.8"
```

Comment:
these and other version added should be 2.9

Reply:
Thanks! They should not be there, since the module already has `version_added: "2.9"`. I removed them both.